server/src/main/java/org/elasticsearch/action/admin/cluster/settings/ClusterUpdateSettingsResponse.java

```diff
@@ -20,16 +20,21 @@
 package org.elasticsearch.action.admin.cluster.settings;
 
 import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.common.ParseField;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.xcontent.ObjectParser;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentParser;
 
-import java.io.IOException;
+import java.io.IOException;;
 
 /**
  * A response for a cluster update settings action.
  */
-public class ClusterUpdateSettingsResponse extends AcknowledgedResponse {
+public class ClusterUpdateSettingsResponse extends AcknowledgedResponse implements ToXContentObject {
```

Comment:
shall we move `RestClusterUpdateSettingsAction` to use `RestToXContentListener`?